Harvest.Core/Data/DbInitializer.cs

```diff
@@ -25,10 +25,10 @@ public async Task Initialize(bool recreateDb)
 
             //Make sure roles exist
             await CheckCreateRole(Role.Codes.System);
-            await CheckCreateRole(Role.Codes.Admin);
             await CheckCreateRole(Role.Codes.FieldManager);
             await CheckCreateRole(Role.Codes.Supervisor);
             await CheckCreateRole(Role.Codes.Worker);
+            await CheckCreateRole(Role.Codes.PI);
```

Comment:
I don't think this is an actual role. Shouldn't be in the DB.

Reply:
Actually ignore this. Sounds like we are going in this direction

Reply:
no i agree here, I am not sure it should be in the DB or an actual role